### PR TITLE
M2-4007: added coverage for Condition

### PR DESCRIPTION
**Pull request by muhurov**

### CODE REVIEW

**Review by alex-i-e — APPROVED** [review]

File: src/modules/Builder/components/ConditionRow/Condition/Condition.test.tsx [hunk 1]
```diff
@@ -0,0 +1,380 @@
...
+  mockedMultiSelectFormValues,
+  mockedSingleSelectFormValues,
+  mockedSliderFormValues,
+} from 'shared/mock';
+import { getEntityKey } from 'shared/utils';
+import { ConditionRowType } from 'modules/Builder/types';
+
+import { renderWithAppletFormData } from 'shared/utils/renderWithAppletFormData';
+import { ConditionType } from 'shared/consts';
+import { getItemOptions, getValueOptionsList } from '../ConditionRow.utils';
```
Comment (wildkiomi) [inline src/modules/Builder/components/ConditionRow/Condition/Condition.test.tsx:18]: empty line

Comment (muhurov) [reply src/modules/Builder/components/ConditionRow/Condition/Condition.test.tsx:18]: fixed

File: src/shared/utils/renderWithAppletFormData.tsx [hunk 2]
```diff
@@ -0,0 +1,39 @@
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-nocheck
+import { ReactNode } from 'react';
+import { FormProvider, useForm } from 'react-hook-form';
+
+import { mockedAppletFormData } from 'shared/mock';
+import { ExtendedRenderOptions } from 'redux/store';
+
+import { AppletFormValues } from 'modules/Builder/types';
```
Comment (kdanilenka32) [inline src/shared/utils/renderWithAppletFormData.tsx:8]: space

Comment (muhurov) [reply src/shared/utils/renderWithAppletFormData.tsx:8]: fixed

**Review by alex-i-e — APPROVED** [review]

**Comment by alex-i-e** [comment]
This PR is almost the same with https://github.com/ChildMindInstitute/mindlogger-admin/pull/1163/files,
it's difficult to detect the changes 👀 , maybe it's better to change the target branch?

**Review by alex-i-e — APPROVED** [review]

**Review by wildkiomi — APPROVED** [review]

**Pull request merged**